Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Class traits

#### Changelog:
- Added traits (5 levels each - Novice/Journeyman/Expert/Master/Legendary) for every magic lifestyle (Priest/Shadow Priest/Warlock/Arcanist/Druid/Necromancer/Pyromancer/Hydromancer/Aeromancer/Geomancer) with opinions (based on character's faith doctrine) about the character with the trait - both criminal/shunned (the same for every level) and positive (increasing with each level). You can find all traits in the in-game encyclopedia and/or by spending perks in corresponding lifestyles.
- Added 1 additional tree for each elemental lifestyle


#### How to test:
- Check if new tooltip in magic lifestyles about traits (near the lifestyle exp progressbar) works correctly.
- Check if traits and its acquisition works correctly.
